.github/CONTRIBUTING.md

```diff
@@ -6,16 +6,17 @@ We accept pull requests for bug fixes and features where we've discussed the app
 
 Please do:
 
-* Check existing issues to verify that the [bug][bug issues] or [feature request][feature request issues] has not already been submitted.
+* Check existing issues to verify that an existing [bug][bug issues] or [feature request][feature request issues] issue does not already exist for the same problem or feature.
 * Open an issue if things aren't working as expected.
 * Open an issue to propose a significant change.
+* Open an issue to propose a design for an issue labelled [`needs-design` and `help wanted`][needs design and help wanted], following the [proposing a design guidelines](#proposing-a-design).
 * Open a pull request to fix a bug.
 * Open a pull request to fix documentation about a command.
 * Open a pull request for any issue labelled [`help wanted`][hw] or [`good first issue`][gfi].
 
 Please avoid:
 
-* Opening pull requests for issues marked `needs-design`, `needs-investigation`, or `blocked`.
+* Opening pull requests for issues marked `needs-investigation`, or `blocked`.
```

Comment:
Why remove `needs-design`, here? The instructions below say "open an issue" to propose designs, so presumably we'd still want folks to avoid opening PRs for `needs-design` issues

Reply:
Excellent point ✨  Let's put that back...
```suggestion
* Opening pull requests for issues marked `needs-design`, `needs-investigation`, or `blocked`.
```